webpack/assets/javascripts/react_app/redux/API/APIReducer.js

```diff
@@ -36,6 +36,12 @@ const apiReducer = (state = initialState, { type, key, payload, response }) => {
           status: ERROR,
         },
       });
+    case UPDATE:
```

Comment:
#TODO: should be replaced by https://github.com/theforeman/foreman/pull/8301

Reply:
Is this done?

Reply:
^^ :)

Reply:
yes, this is a small fix